### PR TITLE
Added Verner Methods

### PR DESCRIPTION
**Pull request by ParamThakkar123**
## Checklist

- [X] Appropriate tests were added
- [X] Any code changes were done in a way that does not break public API
- [ ] All documentation related to code changes were updated
- [X] The new code follows the
  [contributor guidelines](https://github.com/SciML/.github/blob/master/CONTRIBUTING.md), in particular the [SciML Style Guide](https://github.com/SciML/SciMLStyle) and
  [COLPRAC](https://github.com/SciML/COLPRAC).
- [ ] Any new documentation only uses public API

### CODE REVIEW

**Comment by ParamThakkar123** [comment]
@ChrisRackauckas Should I also shift Tsit, BS5, AutoVern methods to the OrdinaryDiffEqVerner lib ?? They are almost always defined together in some files

**Comment by ChrisRackauckas** [comment]
Make an explicit RK group for those, though Tsit5 should be in its own because it's very popular so it should have the lowest dependency setup.

**Comment by ParamThakkar123** [comment]
> @ChrisRackauckas Should I also shift Tsit, BS5, AutoVern methods to the OrdinaryDiffEqVerner lib ?? They are almost always defined together in some files

These methods are defined together in composite_algs.jl file. What should be done for that ??

**Comment by ChrisRackauckas** [comment]
The default algorithm will need to move to an OrdinaryDiffEqDefault which depends on everything it uses in the default algorithm.

**Comment by ChrisRackauckas** [comment]
The default solver needs to be its own thing. For now, just but the imports required for it before its definition.

**Comment by ParamThakkar123** [comment]
What changes must be made here ?? I will make those

File: lib/OrdinaryDiffEqDefault/src/composite_algs.jl [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ### AutoSwitch
 ### Designed to switch between two solvers, stiff and non-stiff
 
+using OrdinaryDiffEq: Vern6, Vern7, Vern8, Vern9
```
Comment (ChrisRackauckas) [inline lib/OrdinaryDiffEqDefault/src/composite_algs.jl:4]: Autoswitch itself should live in the core package. 73-79 should live with the methods that use.  The default package only needs 83 onwards.

**Comment by ParamThakkar123** [comment]
@ChrisRackauckas I am getting a @fold error here. I saw that it's present in MacroTools. I tried to include it but it made no difference.

**Comment by ParamThakkar123** [comment]
@ChrisRackauckas Meanwhile I am done with Rosenbrock methods.

**Comment by ParamThakkar123** [comment]
@ChrisRackauckas There are some precision errors now. Due to which the tests are failing

File: lib/OrdinaryDiffEqFeagin/src/alg_utils.jl [hunk 2]
```diff
@@ -3,4 +3,4 @@ alg_order(alg::Feagin12) = 12
 alg_order(alg::Feagin14) = 14
 
 alg_adaptive_order(alg::Feagin10) = 8
-alg_adaptive_order(alg::Feagin14) = 12
\ No newline at end of file
+alg_adaptive_order(alg::Feagin14) = 12
```
Comment (ChrisRackauckas) [inline lib/OrdinaryDiffEqFeagin/src/alg_utils.jl:6]: don't reformat all of the other ones. Your formatting here is incorrect. It's already corrected on master.

**Comment by ChrisRackauckas** [comment]
For some reason CI didn't trigger, but that should be good to go with the last change.

File: src/OrdinaryDiffEq.jl [hunk 3]
```diff
@@ -410,7 +410,8 @@ export constructDormandPrince
 
 export FunctionMap, Euler, Heun, Ralston, Midpoint, RK4, ExplicitRK, OwrenZen3, OwrenZen4,
        OwrenZen5,
-       BS3, BS5, DP5, Tsit5, DP8, TanYam7, TsitPap8, CompositeAlgorithm, Anas5, RKO65, FRK65, PFRK87,
+       BS3, BS5, DP5, Tsit5, DP8, TanYam7, TsitPap8, CompositeAlgorithm, Anas5, RKO65,
+       FRK65, PFRK87,
```
Comment (ChrisRackauckas) [inline src/OrdinaryDiffEq.jl:414]: are you sure you're on the right formatter version?

Comment (ParamThakkar123) [reply src/OrdinaryDiffEq.jl:414]: I am using this version of JuliaFormatter

[98e50ef6] JuliaFormatter v1.0.56

**Comment by ParamThakkar123** [comment]
@ChrisRackauckas I started a PR for Rosenbrock Methods #2278

**Pull request merged**